vagrant/boxes.d/99-local.yaml.example

```diff
@@ -10,6 +10,20 @@ centos7-katello-devel:
       foreman_devel_github_push_ssh: True
       katello_devel_github_username: <REPLACE ME>
 
+centos8-devel:
+  primary: true
+  box: centos8
+  disk_size: 40
```

Comment:
should this be instead set on the `centos8` box directly, as it will also be required for the pipeline boxes?